plugins/rest-plugin/src/main/java/ru/sbtqa/tag/stepdefs/ru/ApiStepDefs.java

```diff
@@ -54,47 +56,56 @@ public void validate(String rule) {
      * {@inheritDoc}
      */
     @Override
+    @И("^(?:пользователь |он )?проверяет ответ$")
+    public void validate() {
+        super.validate();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
     @И("^система возвращает ответ \"([^\"]*)\" с параметрами:?$")
-    public void validateTable(String rule, DataTable dataTable) {
-        super.validateTable(rule, dataTable);
+    public void validate(String rule, DataTable dataTable) {
+        EndpointContext.getCurrentEndpoint().validate(rule, dataTable);
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     @And("^(?:пользователь |он )? заполняет запрос \"([^\"]*)\"$")
-    public void fillRequest(String title) {
-        super.fillRequest(title);
+    public ApiSteps fill(String title) {
+        return super.fill(title);
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     @And("^(?:пользователь |он )? добавляет (query|header|body) параметр с именем \"([^\"]*)\" и значением \"([^\"]*)\"$")
-    public void addParameter(String parameterType, String name, String value) {
-        super.addParameter(parameterType, name, value);
+    public ApiSteps add(String parameterType, String name, String value) {
+        return super.add(parameterType, name, value);
     }
 
     /**
      * {@inheritDoc}
      */
-    @Override
     @And("^(?:пользователь |он )? добавляет (query|header|body) параметры")
-    public void addParameters(String parameterType, DataTable dataTable) {
-        super.addParameters(parameterType, dataTable);
+    public ApiSteps add(String parameterType, DataTable dataTable) {
+        ParameterType type = ParameterType.valueOf(parameterType.toUpperCase());
+        Map<String, String> dataMap = dataTable.asMap(String.class, String.class);
```

Comment:
How about to make transformer utility method with following signature:
public static Map<String,String> toMap(DataTable dataTable) and use it simple like:
`super.add(type, toMap(dataTable));`